enterprise/backend/src/metabase_enterprise/transforms_python/python_runner.clj

```diff
@@ -194,6 +198,7 @@
                             (let [filtered-col-meta (m/index-by :name fields-meta)
                                   col-names         (map :name cols-meta)
                                   filtered-rows     (eduction (map (fn [row]
+                                                                     (throw-if-cancelled cancel-chan)
```

Comment:
Oops, I was gonna remove this. It could be interpreted as 'better' to keep it, as it aborts the file write if its unflushed. But it just makes things more confusing and its not strictly needed.

Reply:
Yep, agreed is not worth, fixed 77554771006eda2eb2a7a423209d15a8f86c595c